.buildkite/build-start-operator.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [ "$IS_FROM_RAY_RELEASE_AUTOMATION" = 1 ]; then
+    helm install kuberay/kuberay-operator --version 1.3.0;
```

Comment:
Update "Step 3" in https://github.com/ray-project/kuberay/blob/master/docs/development/release.md. Perhaps add a new bullet point between the following two bullet points.

<img width="1158" alt="image" src="https://github.com/user-attachments/assets/6df57acb-1a75-421c-82e1-6dc2d6441c24" />